### DISCUSSION TITLE
Possibility to change Download target folder?

### DISCUSSION THREAD

**Discussion by p1nkyy** [uazo/bromite-buildtools#213] [Q&A] [answered]
Hello, I just made the switch from Bromite to Chromite and I'm pretty sure in Bromite we can change the Download target folder. Is this possible to be done in Chromite? Thanks in advance for any answer.

**Comment by uazo** [comment, marked as answer]
> I'm pretty sure in Bromite we can change the Download target folder

Try it and let us know.

**Reply by p1nkyy** [reply]
My bad, turned out cannot. Sorry!
